js/bibox.js

```diff
@@ -422,6 +424,124 @@ module.exports = class bibox extends Exchange {
         return this.parseBalance (result);
     }
 
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let symbol = undefined;
+        if (code !== undefined) {
+            const currency = this.currency (code);
+            symbol = currency['id'];
+        }
+        let size = (limit) ? limit : 100;
+        let response = await this.privatePostTransfer ({
+            'cmd': 'transfer/transferInList',
+            'body': this.extend ({
+                'search': symbol,
```

Comment:
If this thing is required, then the `code` cannot be undefined. In that case we should throw an `ArgumentsRequired` exception, for example: https://github.com/ccxt/ccxt/blob/master/js/coinex.js#L773

Reply:
code can be left empty have changed the request body structure